build/docker/analytics/init-starrocks-database.sh

```
@@ -0,0 +1,52 @@
+if mysql -h starrocks-fe -P 9030 -u root < /init-create-routine-load.sql 2>/dev/null; then
+  echo -e 'Successfully created routine loads'
+else
+  echo -e 'Routine loads may already exist, continuing...'
+fi
+
+sleep 10s
+echo -e 'Checking and resuming routine loads if needed'
+mysql -h starrocks-fe -P 9030 -u root -e 'show routine load from yorkie\G'
+
+routine_loads=(yorkie.user_events yorkie.channel_events yorkie.session_events)
+for routine_load in ${routine_loads[@]}; do
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)